src/App/Resources/AppResources.resx

```diff
@@ -1390,11 +1397,14 @@
   <data name="ShareItem" xml:space="preserve">
     <value>Share Item</value>
   </data>
+  <data name="MoveToOrganization" xml:space="preserve">
+    <value>Move to Organization</value>
```

Comment:
This title is really snug in a few of those iOS screenshots, and will probably exceed the available width in some translated versions.  That being said, I can't come up with a shorter way to convey 'move to organization' unless we start using 'Org' everywhere which isn't great.  Any thoughts?

Reply:
![image](https://user-images.githubusercontent.com/59324545/122477117-3f0faa00-cf95-11eb-893d-98360c94a524.png)

Reply:
The other verb I pitched was `Transfer`. This might be a good reason to use it.

That being said. These images were taken using an ipod touch. Even on the 8, there's quite a bit more room:
<img width="380" alt="image" src="https://user-images.githubusercontent.com/18214891/122483567-7e8bc580-cf98-11eb-8f82-d8ba86df2fe5.png">